source/darwin/nw_socket.c

```diff
@@ -412,19 +412,28 @@ static bool s_validate_event_loop(struct aws_event_loop *event_loop) {
     return event_loop && event_loop->vtable && event_loop->impl_data;
 }
 
-static void s_set_event_loop(struct aws_socket *aws_socket, struct aws_event_loop *event_loop) {
+static bool s_set_event_loop(struct aws_socket *aws_socket, struct aws_event_loop *event_loop) {
```

Comment:
style: don't use bool for error report.

use int, AWS_OP_SUCCESS and AWS_OP_ERR for consistence. 

Otherwise, when we check for error, some place need to check `if false` and the other places need to check `if not zero`, which is easy to mess up.